helm/vertical-pod-autoscaler-app/values.yaml

```diff
@@ -1,189 +1,85 @@
-# Default values for vertical-pod-autoscaler.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
-# imagePullSecrets -- A list of image pull secrets to be used for all pods
-imagePullSecrets: []
-# priorityClassName -- To set the priorityclass for all pods
-priorityClassName: ""
-# nameOverride -- A template override for the name
 nameOverride: ""
-# fullnameOverride -- A template override for the fullname
 fullnameOverride: ""
```

Comment:
Beware: the definition in _helpers won't see this values when working with the subchart templates.
You may want to set these in `.vertical-pod-autoscaler` values, and do a `coalesce` like I did here with loki: https://github.com/giantswarm/loki-app/blob/master/helm/loki/templates/_helpers.tpl#L6

Reply:
I had originally bot `fullnameOverride` and `nameOverride` both in the global section as well as in the `vertical-pod-autoscaler` one but I eventually deleted the ones in the latter. 
I will put them back as well as creating the `coalesce` in the helpers.

Reply:
Done